changelog.txt

```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.67 - released xxx 2022
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* The backend code for MIDI playback has been changed and real pause logic has been
+  re-enabled. Please be aware that things such as application volume and audio quality
+  may change depending on your configuration from ≤0.66
+* The original demo gameplay video now plays when the CorsixTH main menu is idle (this
+  can be disabled in the configuration file)
+* File browser menus now have an OK button (at last!)
+* Selecting an existing file in the Save Game/Map browsers will now update the text
+  input box to that file name
+
+# Changes
+* Handymen now always stay in their assigned plots, except when going to rest
+* Staff members now have a maximum salary, which will be configurable in custom levels
+* The game will now always use the GP’s choice of next diagnosis room for the patient,
+  if available
+* Queueing patients are now shown when hovering a door
+* Epidemics can no longer fail because an infected patient left before the epidemic was
+  revealed
+* Balance changes to drug improvement research
+* Check for updates can now be disabled in Settings
+* “Pause” now shows up when the “Build while pause” setting is on and the game is paused.
+* The gamelog file now includes a timestamp
+* Many, many, refactors of the code to make it better to understand for future developers
+
+# Translations
+* Chinese Traditional and Simplified translations have been updated. Thanks @lokpro
+* Russian has been updated. Thanks Matt1
```

Comment:
Why @ for lokpro but not Matt1?

Reply:
I don't believe Matt1 has a github account. That is just their Discord username. I can add Discord in brackets if needed.

Reply:
No it's fine. Thanks.